crates/linking/src/lib.rs

```diff
@@ -312,7 +312,12 @@ mod tests {
                 .paths(paths)
                 .ephemeral()
                 .no_artifacts()
-                .build(MultiCompiler { solc: Some(SolcCompiler::Specific(solc)), vyper: None })
+                .build(MultiCompiler {
```

Comment:
I think solidity linking will not work for us. Please explain

Reply:
This part was added because MultiCompiler requires me to pass resolc